cotainr/util.py

```diff
@@ -160,3 +166,22 @@ def _print_and_capture_stream(*, stream_handle, print_dispatch):
         captured_stream.append(line)
 
     return captured_stream
+
+
+def answer_yes(input_text):
+    """
+    Print text and compare the input given to "yes"
+
+    Parameters
+    ----------
+    input_text : str
+        a string to be printed for verification by the user
+
+    Returns
+    -------
+    answer_is_yes : boolean
+        A boolean indicating whether or not the answer is yes
+    """
+    answer = input(input_text).lower()
+    answer_is_yes = answer == "yes"
```

Comment:
This feels pretty user-hostile.

If the user typos `yse`, it'll be the same thing as having answered `no`.

If the only allowed values should be `yes` and `no` (case-insensitive), I feel this should loop until the user enters one of the allowed values.

Reply:
I think that's a fair point - it is a bit harsh to exit and ask the user to start allover if they mistype "yes". And, yes, I know that's what we have done so far - mainly because our initial work on this was somewhat quick-and-dirty. We definitely want the user to clearly answer "yes" before we proceed, but acknowledging a bit of scope creep for this task, I also think we should implement a loop to require either a "yes" or "no" as suggested by @akx.

Reply:
@Chroxvi fair. I'll have a look at making more like the usual linux yes/no handling then.

Reply:
The handling has been updated